docs/sources/operations/storage/logs-deletion.md

```diff
@@ -4,10 +4,11 @@ weight: 60
 ---
 # Log Entry Deletion
 
-<span style="background-color:#f3f973;">Log entry deletion is experimental. It is only supported for the BoltDB Shipper index store.</span>
+<span style="background-color:#f3f973;">Log entry deletion is only supported for the BoltDB Shipper index store.</span>
```

Comment:
I strongly suggest that this color only be used for the experimental feature note. I don't think that you need a color at all.  This suggestion removes the highlight color and introduces emphasis with italics.  I hope that others doing reviews will provide their opinions on how to note something like this.  Note that too many "Notes" tend to cause the reader to skip over them, the opposite of what you want.

```suggestion
Log entry deletion is supported _only_ for the BoltDB Shipper index store.
```